### PR TITLE
Adding wdio-eslinter-service to auto-detect missing require imports i…

### PR DESCRIPTION
**Pull request by jamesmortensen**
…n JavaScript tests prior to running tests

## Proposed changes

Adding a link to wdio-eslinter-service, which runs eslint on test scripts prior to running the tests.  This is just a documentation change to link to this service.

## Types of changes

The service being linked to runs eslint on the codebase and prevents execution if errors are found.  The default configuration is lightweight and simply looks for broken require statements but can be expanded by the user to add more eslint rules.

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation update

## Checklist

- [X] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

I created the service because we would find incorrect paths in the require statements in the middle of a test run. This prevents that from happening.

### CODE REVIEW

**Review by christian-bromann — COMMENTED** [review]
Would you mind also adding it here: https://github.com/webdriverio/webdriverio/blob/main/packages/wdio-cli/src/constants.ts#L92-L126, so that the CLI wizard detects it?

**Comment by jamesmortensen** [comment]
Hi @christian-bromann I added it in, but I wasn't sure how to test it due to missing documentation https://github.com/webdriverio/webdriverio/issues/6876 on how to link directly to the wdio-cli package.

I copied the same format that I saw in other places.  Please let me know what else I need to do to test/validate this.  Thank you.

File: scripts/docs-generation/3rd-party/services.json [hunk 1]
```diff
@@ -123,5 +123,12 @@
...
     "githubUrl": "https://github.com/elaichenkov/wdio-wait-for",
     "npmUrl": "https://www.npmjs.com/package/wdio-wait-for"
+  },
+  {
+    "packageName": "wdio-eslinter-service",
+    "title": "Auto-detect missing imports w/eslint",
+    "githubUrl": "https://github.com/jamesmortensen/wdio-eslinter-service",
+    "npmUrl": "https://www.npmjs.com/package/wdio-eslinter-service"
   }
+
```
Comment (christian-bromann) [inline scripts/docs-generation/3rd-party/services.json:133]:
```suggestion
```

**Review by christian-bromann — APPROVED** [review]
👍  thanks

**Pull request merged**